src/modules/fetchGrade.js

```diff
@@ -3,7 +3,7 @@ import showGrade from './showGrade.js';
 class Grades {
   constructor() {
     this.tbody = document.querySelector('.body-area');
-    this.url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/:id/scores/';
+    this.url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/troosleaderboad/scores/.';
```

Comment:
- [Optional] KIndly, Make sure you have used the correct endpoint and given it the correct data to your system as it is the main functionality this project👍